### PR TITLE
feat(api): seeding for env

### PR DESCRIPTION
**Pull request by fabfab-kc**
Seeds LPAs from a different list depending on the environment

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: appeals/api/src/database/seed/seed-training.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+		await seedLPAs(databaseConnector, localPlanningDepartmentList);
+	} catch (error) {
+		console.error(error);
+		throw error;
+	} finally {
+		await databaseConnector.$disconnect();
+	}
+};
+
+await (process.env.NODE_ENV === 'training' ? seedTraining() : null);
```
Comment (beejjacobs) [inline appeals/api/src/database/seed/seed-training.js:24]: Not sure we need to set/check `NODE_ENV` (I know copied from the prod one, but I wasn't sure why that was there either!

File: appeals/api/src/database/seed/LPAs/prod.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/** @typedef {Omit<import('@pins/appeals.api').Schema.LPA, 'id'>} LPA */
```
Comment (beejjacobs) [inline appeals/api/src/database/seed/LPAs/prod.js:1]: We should use the create Prisma type here, it'll be something like

import('<db-client>').Prisma.LpaCreateInput

File: appeals/api/src/database/seed/seed-training.js (hunk 1 above)
Comment (fabfab-kc) [reply appeals/api/src/database/seed/seed-training.js:24]: yeah wasn't so sure about that... those are passed to the npm scripts with CROSS-ENV

File: azure-pipelines-db-seed-appeals.yml
Comment (beejjacobs) [inline azure-pipelines-db-seed-appeals.yml:1]: `keyVaultName` needs to end in just `train` for the training env

**Review by beejjacobs — APPROVED** [review]
LGTM.

Might be worth running the pipeline to check for syntax errors etc...

**Pull request merged**